package.json

```diff
@@ -37,6 +37,7 @@
     "@trussworks/react-uswds": "2.4.1",
     "classnames": "2.3.1",
     "graphql": "16.0.1",
+    "mongodb": "^3.5.9",
```

Comment:
If you pre-emptively pin this version to the one that's actually installed (looks like `3.7.3` according to the yarn.lock file) then Renovate bot won't open a PR to pin it